app/code/Magento/Checkout/view/frontend/web/js/model/shipping-save-processor/payload-extender.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright © 2013-2017 Magento, Inc. All rights reserved.
```

Comment:
Looks like this is an artifact form the previous copyright update. 
Please change it to the one without specific year like in all other code files.